### PR TITLE
Exception homework

### PR DESCRIPTION
**Pull request by dacian-cioc**

### CODE REVIEW

File: src/L12/homework/FileSubmissionDemo.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+//they get 0 points. If the file submitted is null or an empty string, an exception
+//should be thrown and you should give the student -1 points. What kind of
+//exception - your own custom exception!
+//Each student has: name, array with 5 files and score.
+//In the main(), create at least 3 students in order to test out your work. If an
+//exception is caught, make sure to print out appropriate error message.
+
+package L12.homework;
+
+class InvalidFileException extends Exception {
```
Comment (KaloyanTsvetkov) [inline src/L12/homework/FileSubmissionDemo.java]: Please use separate files for all classes.

File: src/L12/homework/FileSubmissionDemo.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        super(message);
+    }
+}
+
+class Student {
+    private String name;
+    private String[] files;
+    private int score;
+
+    public Student(String name, String[] files) {
```
Comment (KaloyanTsvetkov) [inline src/L12/homework/FileSubmissionDemo.java]: Each student has: name, array with 5 files and score.
So the constructor should set only the name of the student, score = 0 and to initialize the array with lengh [5] (files = new String[5])
For adding a file you should have separatye method.

If you want to keep this constructor:
public Student(String name, String[] files)

please create a constructor chain.

Comment (dacian-cioc) [reply src/L12/homework/FileSubmissionDemo.java]: updated

File: src/L12/homework/FileSubmissionDemo.java (hunk 1 above)
Comment (dacian-cioc) [reply src/L12/homework/FileSubmissionDemo.java]: updated

**Review by KaloyanTsvetkov — APPROVED** [review]

**Pull request merged**